sopel/tools/target.py

```diff
@@ -265,13 +292,13 @@ def is_admin(self, nick):
             sensibly if only ``+v`` (voice) and ``+o`` (op) modes exist.
 
         """
-        return self.privileges.get(Identifier(nick), 0) & privileges.ADMIN
+        nick_id = self.make_identifier(nick)
+        return bool(self.privileges.get(nick_id, 0) & privileges.ADMIN)
```

Comment:
Double take alert… I thought this was touching the database (!) for some unknown reason. This might reduce the WTF factor for future developers trekking through this part of the code:

```suggestion
        identifier = self.make_identifier(nick)
        return bool(self.privileges.get(identifier, 0) & privileges.ADMIN)
```

Or one could K.I.S.S. instead (if type-checking doesn't get mad about this):

```suggestion
        nick = self.make_identifier(nick)
        return bool(self.privileges.get(nick, 0) & privileges.ADMIN)
```

_Consider this same note applied to all the methods below using the same local variable name, `nick_id`._

Reply:
Type checking will get extra mad at this, yes.

Reply:
So I hear that the type-checker doesn't like changing the type of a variable, fine. First version, then? I'm just looking for an alternative name that isn't `nick_id`. 😺

Reply:
Yup, `identifier` is good!